src/filterbutton.js

```diff
@@ -0,0 +1,47 @@
+//---------------------------------Filter Buttons Starts----------------------------------------------------------//
+const filterBtns = document.querySelectorAll('.filter-btn');
+
+function displayMenuButtons() {
```

Comment:
Probably you would need to export this func 
```suggestion
export const displayMenuButtons = () => {
```